packages/app/plugin/src/ios/googleServicesPlist.ts

```diff
@@ -0,0 +1,60 @@
+import { ConfigPlugin, withXcodeProject, XcodeProject } from '@expo/config-plugins';
+import { getSourceRoot } from '@expo/config-plugins/build/ios/Paths';
```

Comment:
Can use `IOSConfig.Paths.getSourceRoot()` so you aren't reaching into the package internals.